src/software/simulation/physics/physics_world.h

```diff
@@ -84,12 +202,12 @@ class PhysicsWorld
 
     std::unique_ptr<SimulationContactListener> contact_listener;
 
-    // Abstractions of objects in the world
-    std::shared_ptr<PhysicsBall> physics_ball;
+    // TODO: is the best way to handle the field? Just check it everywhere to maintain
+    // no nullptr invariant? Needs to be a pointer so we can destroy the underlying
+    // object when we set a new one, since the PhysicsField has no copy or assignment
+    // operator
```

Comment:
Where or why are we setting new Fields?

Reply:
the `setField` function

Reply:
Will get rid of this

Reply:
as per offline discussion made the field a member variable and removed setter functions.

Reply:
fun fact: `World` won't have a `setField` function after #1413